packages/flutter/lib/src/foundation/_isolates_io.dart

```diff
@@ -83,7 +83,16 @@ class _IsolateConfiguration<Q, R> {
   FutureOr<R> apply() => callback(message);
 }
 
-Future<void> _spawn<Q, R>(_IsolateConfiguration<Q, FutureOr<R>> configuration) async {
+/// The spawn point MUST guarantee only one result event is sent through the
+/// [SendPort.send] be it directly or indirectly i. e. [Isolate.exit].
```

Comment:
```suggestion
/// [SendPort.send] be it directly or indirectly i.e. [Isolate.exit].
```

Reply:
Fixed and thank you for reviewing this PR.